domain/src/main/java/nl/tehdreamteam/se42/domain/Message.java

```diff
@@ -0,0 +1,12 @@
+package nl.tehdreamteam.se42.domain;
+
+/**
+ * @author Lesley
+ */
+public class Message {
+
+    public Message() {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1186)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Hide this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1118)